### PR TITLE
some more constants

### PR DESCRIPTION
**Pull request by Megamouse**
Please tell me I used them in the correct places.
Also, min stack size seems to have been wrong. Unless that was intentional

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_process.h [hunk 1]
```diff
@@ -28,6 +28,50 @@ enum : u32
...
+constexpr auto SYS_PROCESS_PARAM_SECTION_NAME = ".sys_proc_param";
+
+enum
+{
+	SYS_PROCESS_PARAM_INVALID_PRIO = -32768,
+};
+
+enum : u32
+{
+	SYS_PROCESS_PARAM_INVALID_STACK_SIZE = 0xffffffffUL,
```
Comment (elad335) [inline rpcs3/Emu/Cell/lv2/sys_process.h:51]:
```suggestion
	SYS_PROCESS_PARAM_INVALID_STACK_SIZE = 0xffffffff,
```

File: rpcs3/Emu/Cell/lv2/sys_process.h [hunk 2]
```diff
@@ -28,6 +28,50 @@ enum : u32
 	SYS_EVENT_FLAG_OBJECT            = 0x98,
 };
 
+enum : u64
+{
+	SYS_PROCESS_PRIMARY_STACK_SIZE_32K  = 0x0000000000000010ULL,
```
Comment (elad335) [inline rpcs3/Emu/Cell/lv2/sys_process.h:33]:
```suggestion
	SYS_PROCESS_PRIMARY_STACK_SIZE_32K  = 0x0000000000000010,
```

File: rpcs3/Emu/Cell/PPUModule.cpp [hunk 3]
```diff
@@ -333,7 +333,7 @@ static void ppu_initialize_modules(ppu_linkage_info* link)
 
 				if (!next || (end >> 16 != alloc_addr >> 16))
 				{
-					alloc_addr = vm::alloc(0x10000, vm::main);
+					alloc_addr = vm::alloc(SYS_PROCESS_PARAM_MALLOC_PAGE_SIZE_64K, vm::main);
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUModule.cpp:336]: True they have the same value, but this is not the meaning of the enum.

File: rpcs3/Emu/Cell/PPUModule.cpp [hunk 4]
```diff
@@ -322,9 +322,9 @@ static void ppu_initialize_modules(ppu_linkage_info* link)
 			ppu_loader.trace("** &0x%08X: %s (size=0x%x, align=0x%x)", variable.first, variable.second.name, variable.second.size, variable.second.align);
 
 			// Allocate HLE variable
-			if (variable.second.size >= 0x10000 || variable.second.align >= 0x10000)
+			if (variable.second.size >= SYS_PROCESS_PARAM_MALLOC_PAGE_SIZE_64K || variable.second.align >= SYS_PROCESS_PARAM_MALLOC_PAGE_SIZE_64K)
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUModule.cpp:325]: Same

File: rpcs3/Emu/Cell/PPUModule.cpp [hunk 5]
```diff
@@ -322,9 +322,9 @@ static void ppu_initialize_modules(ppu_linkage_info* link)
...
 			{
-				variable.second.addr = vm::alloc(variable.second.size, vm::main, std::max<u32>(variable.second.align, 0x10000));
+				variable.second.addr = vm::alloc(variable.second.size, vm::main, std::max<u32>(variable.second.align, SYS_PROCESS_PARAM_MALLOC_PAGE_SIZE_64K));
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUModule.cpp:327]: Same.

**Review by elad335 — CHANGES_REQUESTED** [review]

File: rpcs3/Emu/Cell/PPUModule.cpp (hunk 3 above)
Comment (Megamouse) [reply rpcs3/Emu/Cell/PPUModule.cpp:336]: yeah. those were the ones I wasn't sure about

**Review by elad335 — APPROVED** [review]

File: rpcs3/Emu/Cell/PPUModule.cpp [hunk 6]
```diff
@@ -1674,16 +1674,16 @@ bool ppu_load_exec(const ppu_exec_object& elf)
...
+	case SYS_PROCESS_PRIMARY_STACK_SIZE_64K:  primary_stacksize =   64 * 1024; break;
+	case SYS_PROCESS_PRIMARY_STACK_SIZE_96K:  primary_stacksize =   96 * 1024; break;
+	case SYS_PROCESS_PRIMARY_STACK_SIZE_128K: primary_stacksize =  128 * 1024; break;
+	case SYS_PROCESS_PRIMARY_STACK_SIZE_256K: primary_stacksize =  256 * 1024; break;
+	case SYS_PROCESS_PRIMARY_STACK_SIZE_512K: primary_stacksize =  512 * 1024; break;
+	case SYS_PROCESS_PRIMARY_STACK_SIZE_1M:   primary_stacksize = 1024 * 1024; break;
 	default:
 	{
-		primary_stacksize = utils::align<u32>(std::clamp<u32>(sz, 0x10000, 0x100000), 4096);
+		primary_stacksize = utils::align<u32>(std::clamp<u32>(sz, SYS_PROCESS_PARAM_STACK_SIZE_MIN, SYS_PROCESS_PARAM_STACK_SIZE_MAX), 4096);
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/PPUModule.cpp:1686]: this basically changes min from 0x10000 to 0x1000

Comment (elad335) [reply rpcs3/Emu/Cell/PPUModule.cpp:1686]: Oh, this was hw tested I think, leave min at 64k for now.

**Pull request merged**